index.html

```diff
@@ -416,65 +416,172 @@ <h2 id="liaisons">
+      Contributor License Agreement (CLA)</a>. 
     </p>
+    
     <p>
-      Specifications created for proposals in the Community Group must use the
+      Specifications created in the Community Group must use the
       <a href=
       "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">
-      W3C Software and Document License</a>.
+      W3C Software and Document License</a>.  All other documents produced 
+      by the group should use that License where possible. 
     </p>
+
+    <p class="remove">
+      {TBD: if CG doesn't use GitHub replace the remaining paragraphs in this 
+      section with: "All Contributions are made on the groups public mail list
+      or public contrib list"} 
+    </p>
+    
+    <p>
+      Community Group participants agree to make all contributions 
+      in the GitHub repo the group is using for the particular document. 
+      This may be in the form of a pull request (preferred), by raising an issue, 
+      or by adding a comment to an existing issue.
+    </p>
+
     <p>
       All Github repositories attached to the Community Group must contain a
-      copy of the <a href="CONTRIBUTING.md">CONTRIBUTING</a> and <a href=
-      "LICENSE.md">LICENSE</a> files.
+      copy of the 
+      <a href="https://github.com/w3c/licenses/blob/master/CG-CONTRIBUTING.md">CONTRIBUTING</a>
+      and <a href=
+      "https://github.com/w3c/licenses/blob/master/CG-LICENSE.md">LICENSE</a> files.
+    </p>
+   <p class="note">Proposed charter deleted last section of this paragraph</p>
+    <p>
+      Note: this Community Group will not use a contrib mailing list for contributions 
+      since all contributions will be tracked via Github mechanisms 
+      (e.g. pull requests). For the same reason, the Community Group 
+      mailing list must not be used for making or discussing substantive contributions
+      to Specifications.
     </p>
+    
+    <h2 id="transparency">
+      Transparency
+    </h2>
+    
     <p>
-      Note: this Community Group will not use a <em>contrib</em> mailing list
-      for contributions since all contributions will be tracked via Github
-      mechanisms (e.g. pull requests).
+      The group will conduct all of its technical work in public. If the group 
+      uses GitHub, all technical work will occur in its GitHub repositories 
+      (and not in mailing list discussions). This is to ensure contributions 
+      can be tracked through a software tool.
     </p>
+  
+   <p class="note">Proposed charter deleted this paragraph</p>  
+    <p>
+      Meetings may be restricted to Community Group participants, but a 
+      public summary or minutes must be posted to the group's public mailing 
+      list, or to a GitHub issue if the group uses GitHub.
+    </p>
+
+    <h2 id="decision">
+      Decision Process
+    </h2>
+    
+    <p>This group will seek to make decisions where there is consensus. 
```

Comment:
I would prefer that PR be formatted consistently using tidy-html5.  For the Presentation API we use the following configuration [1].

The Work Mode page [2] for the Second Screen Working Group has details on setting up tidy-html5.  If it's tricky for you, then it's okay to submit as-is and I can submit a cleanup commit afterwards.

[1] https://github.com/w3c/presentation-api/blob/gh-pages/build/tidyconfig.txt
[2] https://www.w3.org/wiki/Second_Screen/Work_Mode#How_to_Setup_the_Environment